index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <!-- Font Awesome CDN -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <ul class="nav-flex">
+          <li class="nav-link"><a href="#">John Doe</a></li>
+          <li class="nav-link">
+            <a href="#"><i class="fa-solid fa-bars"></i></a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Kindly be noticed that you should end the `header` tag after the `nav` tag ie.
```
<header>
      <nav class="navbar">
        <ul class="nav-flex">
          <li class="nav-link"><a href="#">John Doe</a></li>
          <li class="nav-link">
            <a href="#"><i class="fa-solid fa-bars"></i></a>
          </li>
        </ul>
      </nav>
</header>
```